internal/storage/bucket_handle_test.go

```diff
@@ -467,3 +470,293 @@ func (t *BucketHandleTest) TestUpdateObjectMethodWithMissingObject() {
 
 	AssertTrue(errors.As(err, &notfound))
 }
+
+// Read content of an object and return
+func (t *BucketHandleTest) testObjectContent(ctx context.Context, req *gcs.ReadObjectRequest) (buffer string) {
```

Comment:
This method is just reading the content and not testing the object content. Please rename it accordingly.

Reply:
fixed